package.json

```diff
@@ -11,14 +11,13 @@
   "author": "Frederic Hemberger (https://frederic-hemberger.de/)",
   "license": "MIT",
   "dependencies": {
+    "@hapi/hoek": "^9.0.2",
+    "@hapi/wreck": "^17.0.0",
     "fast-safe-stringify": "^2.0.2",
-    "good-http": "^6.1.3",
-    "good-udp": "4.1.x",
-    "hoek": "^4.1.1",
-    "lodash": "^4.17.10",
-    "wreck": "^12.2.0"
+    "lodash": "^4.17.10"
   },
   "devDependencies": {
+    "@types/node": "^13.1.7",
```

Comment:
Node 13? Should be 8 or 10 I'd think ... since we still have some Node 8 services, I think 8.